modules/core/05-port/types/ibc_legacy_module.go

```diff
@@ -118,22 +118,6 @@ func (im *LegacyIBCModule) OnChanOpenTry(
 	return reconstructVersion(im.cbs, negotiatedVersions)
 }
 
-// reconstructVersion will generate the channel version by applying any version wrapping as necessary.
-// Version wrapping will only occur if the negotiated version is non=empty and the application is a VersionWrapper.
-func reconstructVersion(cbs []ClassicIBCModule, negotiatedVersions []string) (string, error) {
```

Comment:
I just moved this to the bottom of the file since most handlers are not using this.